app/helpers/admin/sidebar_helper.rb

```diff
@@ -14,16 +14,16 @@ def simple_formatting_sidebar(options = {})
     end
   end
 
-  def edition_tabs(_edition, options = {})
+  def edition_tabs(edition, options = {})
     options = { editing: false, history_count: 0, remarks_count: 0 }.merge(options)
     {}.tap do |tabs|
       if options[:editing]
         tabs[:govspeak_help] = "Help"
       end
       tabs[:notes] = ["Notes", options[:remarks_count]]
       tabs[:history] = ["History", options[:history_count]]
-      if @edition.can_be_fact_checked?
-        tabs[:fact_checking] = ["Fact checking", @edition.all_completed_fact_check_requests.count]
+      if edition.can_be_fact_checked?
```

Comment:
Curious! Looks like this snuck in 8 years ago: https://github.com/alphagov/whitehall/commit/186e61b1a35f83ece10b5d91bb977b1544d4dd23#diff-5ede572e8c3d99712827002de6d600eeR10-R17. Interesting how we haven't been referencing the argument `edition` that entire time 😧